soh/src/code/z_actor.c

```diff
@@ -3142,6 +3143,40 @@ int gMapLoading = 0;
 
 Actor* Actor_Spawn(ActorContext* actorCtx, GlobalContext* globalCtx, s16 actorId, f32 posX, f32 posY, f32 posZ,
                    s16 rotX, s16 rotY, s16 rotZ, s16 params) {
+
+    // Hack to remove bats and skulltulas that spawn in graveyard because of removing object dependency.
+    if ((actorId == ACTOR_EN_FIREFLY || (actorId == ACTOR_EN_SW && params == 0)) && globalCtx->sceneNum == SCENE_SPOT02) {
+        return NULL;
+    }
+
+    uint8_t tryRandomizeEnemy = CVar_GetS32("gRandomizedEnemies", 0) && gSaveContext.fileNum >= 0 && gSaveContext.fileNum <= 2;
```

Comment:
@garrettjoecox re: your comment about cc enemy spawns, any ideas of things to add to this line to account for that (or any other situations i'm not thinking of)